test/recipes/15-test_ml_dsa_codecs.t

```diff
@@ -56,7 +56,7 @@ foreach my $alg (@algs) {
         # Compare expected DER public key with DER public key of private
         ok(run(app(['openssl', 'pkey', '-in', $in, '-pubout',
                     '-outform', 'DER', '-out', $der])));
-        ok(!compare($der0, $der),
+        ok(!compare_text($der0, $der),
```

Comment:
I do not think this is correct - these are binary files.

Reply:
Fixed in a new commit.